README.md

```diff
@@ -1 +1,3 @@
-# akka-cluster-management
\ No newline at end of file
+# akka-cluster-management
+
+This repository contains interfaces to interact with an Akka Cluster.
```

Comment:
note: we should explain what bin compat guarantees we give here, should also note the license and gitter chats etc.

Does not have to be done in this PR, even better if we add it afterwards :)